examples/main/main.cpp

```diff
@@ -52,6 +53,8 @@ int main(int argc, char ** argv) {
         return 1;
     }
 
+    fprintf(stderr, "%s: build = %d (%s)\n", __func__, BUILD_NUMBER, BUILD_COMMIT);
```

Comment:
It would be good to have this at least on the perplexity example as well, maybe in all of them.